elections/2023-SC-EU/candidate-nrrso.md

```diff
@@ -0,0 +1,28 @@
+-------------------------------------------------------------
+name: Norris Sam Osarenkhoe
+
+ID: nrrso
+
+info:
+  - affiliation: SVA System Vertrieb Alexander GmbH
+-------------------------------------------------------------
+
+
+## Contributions to Knative
```

Comment:
I am not 100% sure how the headers will look like in Elekto (e.g. https://elections.knative.dev/app/elections/2023-SC/candidates/aliok) but we can update later on.